fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/AuthenticatedRequestAuthenticationConverter.java

```diff
@@ -41,8 +42,8 @@ public Authentication convert(HttpServletRequest request) {
         .orElseGet(
             () ->
                 new AnonymousAuthenticationToken(
-                    "anonymous",
-                    "anonymous",
-                    AuthorityUtils.createAuthorityList("ROLE_ANONYMOUS")));
+                    SpinnakerUsers.ANONYMOUS,
```

Comment:
notes for reviewers -- the new code looks like it does exactly the same thing as the old code, so this is "just" a refactor.